frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/components/SummonsOrderComponent.js

```diff
@@ -175,7 +175,9 @@ const SummonsOrderComponent = ({ t, config, formData, onSelect }) => {
   const selectedParty = useMemo(() => {
     return formData[config.key]?.party != null
       ? {
-          label: [formData[config.key]?.party?.data?.firstName, formData[config.key]?.party?.data?.lastName].filter(Boolean).join(" "),
+          label:
+            [formData[config.key]?.party?.data?.firstName, formData[config.key]?.party?.data?.lastName].filter(Boolean).join(" ") +
+            ` (${formData[config.key]?.party?.data?.partyType})`,
```

Comment:
what happens when `formData[config.key]?.party?.data?.partyType` turns out to be undefined ?

Reply:
addressed